users/models.py

```diff
@@ -23,8 +23,13 @@ def possible_admins(self):
         return self.filter(guardian=None)
 
 
+# This is needed when using a custom User queryset
+class UserManager(OriginalUserManager.from_queryset(UserQuerySet)):
+    pass
+
+
 class User(AbstractUser):
-    objects = BaseUserManager.from_queryset(UserQuerySet)()
+    objects = UserManager()
```

Comment:
So I guess that separate class definition is required and `OriginalUserManager.from_queryset(UserQuerySet)` cannot be used here?

Reply:
Yeah it seems so, without the class at least `makemigrations` fails with

```
ValueError: Could not find manager UserManagerFromUserQuerySet in django.db.models.manager.
Please note that you need to inherit from managers you dynamically generated with 'from_queryset()'.
```

Reply:
Okay then, all good here! :+1: